src/test/ui/const-generics/trait-const-args.rs

```diff
@@ -0,0 +1,31 @@
+// check-pass
+
+#![feature(const_generics)]
+//~^ WARN the feature `const_generics` is incomplete and may cause the compiler to crash
+#![feature(lazy_normalization_consts)]
+//~^ WARN the feature `lazy_normalization_consts` is incomplete and may cause the compiler to crash
+
+struct Const<const N: usize>;
+trait Foo<const N: usize> {}
+
+impl<const N: usize> Foo<{N}> for Const<{N}> {}
+
+fn foo_impl(_: impl Foo<3>) {}
+
+fn foo_explicit<T: Foo<3>>(_: T) {}
+
+fn foo_where<T>(_: T) where T: Foo<3> {}
+
+fn main() {
+    // FIXME this causes a stack overflow in rustc
```

Comment:
```suggestion
    // FIXME(lazy_normalization_consts) this causes a stack overflow in rustc
```
(And below twice.)

Reply:
Afaict this fixme was copy pasted from the test itself, we should just remove these fixmes entirely

Reply:
This already compiles on the currently nightly, moved this file to #72008